lab-jaren/test/game-router-test.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+require('dotenv').load();
+const superagent = require('superagent');
+const expect = require('chai').expect;
+const Game = require('../model/game.js');
+const baseURL = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+describe('testing game router, tests should not leave files in data', function() {
+  let mockData = {
+    title: 'XCOM 2',
+    genre: 'strategy/tactics',
+    developer: '2K Games, Inc.',
+    publisher: 'Firaxis Games, Inc.',
+    platforms: 'Windows, Linux, Mac, PS4, Xbox One',
+    ratingESRB: 'Teen',
+    releaseDate: 'Feb 05, 2016',
+  };
+  describe('testing POST /api/games', function() {
+    after(done => {
+      Game.deleteById(this.tempGameId)
+      .then(() => done())
+      .catch(done);
+    });
+    it('valid input should create a game with status 200', done => {
+      superagent.post(`${baseURL}/api/games`)
+      .send(mockData)
+      .then(res => {
+        expect(res.status).to.equal(200);
```

Comment:
I realize that the ``for (let key in obj)`` loop I told you about last time would not apply here since the date and id fields are different types. 

But you can still get around that with an if statement and some ``toString()`` manipulation. I showed this code to David since he has lots of properties too, so the variable name of ``this.tempBeer`` applies to him, but just apply it to your own model. 

```
When you have a lot of properties to test for, writing all these expect statements can be a pain. I refactored these expect statements to the following code:

```
for(let key in res.body){
          if (key === 'created'){
            res.body[key] = new Date(res.body.created.toString());
          }
          expect(res.body[key].toString()).to.equal(this.tempBeer[key].toString());
        }
```

If you have questions about the logic of the ``for (let key in obj)`` loop, let me know.